dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/SingleDirScanJob.java

```diff
@@ -55,10 +55,11 @@ public Void call() {
         }
       }
       scanCtx.endWalkDir(dir, numFiles, numDirs, accumFileSize);
-    } catch (Exception exn) {
+    } catch (Exception e) {
       LOG.error(
-          "Unexpected exception while scanning HDFS folder " + dir.getPath().toUri().getPath(),
-          exn);
+          "Unexpected exception while scanning HDFS folder '{}'",
+          dir.getPath().toUri().getPath(),
+          e);
```

Comment:
Are you sure this change to this call does what we want it to? The three-argument error() call does not take a Throwable as the third argument. The two-argument does take a Throwable as the second argument.

Reply:
Slf4j will replace the placeholder `{}` with the path and then it will additionally process the exception, because it is the last argument.

https://www.slf4j.org/faq.html#paramException